src/allmydata/test/test_storage.py

```diff
@@ -1,3 +1,13 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
```

Comment:
This should have a docstring that at least states the source file has been ported to Python 3.